### PR TITLE
Tianyi Song_Update roster_names.md

### PR DESCRIPTION
**Pull request by Evanna-Song**
Added my name Tianyi Song to roster_names.md

#7

### CODE REVIEW

File: roster_names.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 | Name | 
-| ---  | 
-| Mitch Phillipson | 
\ No newline at end of file
+|---|
```
Comment (mitchphillipson) [inline roster_names.md:2]: This line is necessary for the table to format correctly

**Comment by mitchphillipson** [comment]
I see a merge conflict, can you correct that. I did this in the recording. Click "Resolve Conflicts" and you'll see code that looks like

```
<<< main
Your Code Here
==========
My code here
>>> main
```

Your job is pick from your code and mine to get the correct format. You will delete the `<<<main`, `=====` and `>>>main` lines and maybe delete other things. Then click `Mark as Resolved` in the upper right when you're done.

**Comment by mitchphillipson** [comment]
Also, when you resolve the conflict, don't remove any names

**Comment by Evanna-Song** [comment]
> I see a merge conflict, can you correct that. I did this in the recording. Click "Resolve Conflicts" and you'll see code that looks like
> 
> ```
> <<< main
> Your Code Here
> ==========
> My code here
> >>> main
> ```
> 
> Your job is pick from your code and mine to get the correct format. You will delete the `<<<main`, `=====` and `>>>main` lines and maybe delete other things. Then click `Mark as Resolved` in the upper right when you're done.

OK, I have Resolved Conflicts and click Mark as Resolved.

**Comment by Evanna-Song** [comment]
#8 into main

**Pull request merged**